tests/Metadata/TargetsMetadataTest.php

```diff
@@ -212,4 +212,68 @@ public function testDuplicateDelegatedRoleNames(): void
         $this->expectExceptionMessage('Delegated role names must be unique.');
         static::callCreateFromJson($json);
     }
+
+    public function testKeysAreSorted(): void
+    {
+        $data = [
+            'signatures' => [],
+            'signed' => [
```

Comment:
Why not have top level keys out of order and assert they are fixed too?

Reply:
Sure, why not? Added an assertion to that effect.